Framework/DataHandling/src/LoadSampleEnvironment.cpp

```diff
@@ -53,6 +53,9 @@ void LoadSampleEnvironment::init() {
       make_unique<FileProperty>("Filename", "", FileProperty::Load, extensions),
       "The path name of the file containing the Environment");
 
+  // scale to use for stl
+  declareProperty("Scale", "cm", "The scale of the stl: m, cm, or mm");
```

Comment:
Should the default not be "m",  the current behaviour?

Reply:
since `LoadSampleShape` is not currently used, it made the most sense to use cm to match other uses of setting shapes.